src/components/charts/shared/StellarValueRange.jsx

```diff
@@ -1,18 +1,16 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { extentFromSet } from '../../../lib/utilities.js';
 import StellarValue from './StellarValue';
 
 class StellarValueRange extends React.PureComponent {
   render() {
     const { type, data } = this.props;
-    const minMax = extentFromSet(data, type);
 
     return (
       <>
-        <StellarValue type={type} value={minMax[0]} />
-        ` – `
-        <StellarValue type={type} value={minMax[1]} />
+        <StellarValue type={type} value={data[0]} />
+        <span> – </span>
```

Comment:
I think the backticks are lingering from a template literal string, they did not look correct encapsulating the dash. 

This component was not in use and I changed some of the functionality. Previously it would extract the range from the dataset using an accessor and a helper method, now it simply uses the only two values in the range which are assumed to be the min/max. We could revert to extracting the min/max to allow passing full arrays.